src/fancy-tools/ellipticalRoiTool.js

```diff
@@ -0,0 +1,382 @@
+/* eslint no-loop-func: 0 */ // --> OFF
+import external from './../externalModules.js';
+import baseAnnotationTool from './../base/baseAnnotationTool.js';
+// State
+import { getToolState } from './../stateManagement/toolState.js';
+import toolStyle from './../stateManagement/toolStyle.js';
+import toolColors from './../stateManagement/toolColors.js';
+// Manipulators
+import drawHandles from './../manipulators/drawHandles.js';
+// Drawing
+import { getNewContext, draw, setShadow, drawEllipse } from './../util/drawing.js';
+import drawLinkedTextBox from './../util/drawLinkedTextBox.js';
+import pointInEllipse from '../util/pointInEllipse.js';
+import calculateEllipseStatistics from '../util/calculateEllipseStatistics.js';
+import calculateSUV from '../util/calculateSUV.js';
+
+
+const cornerstone = external.cornerstone;
+
+const numberWithCommas = (x) => {
```

Comment:
Similar note to `rectangleRoiTool`. This should be moved to a shared location and imported.